source/_common.R

```diff
@@ -76,3 +78,19 @@ get_var = function(name) {
   }
   return (get(name))
 }
+
+# See: https://stackoverflow.com/questions/76732162/is-there-a-way-to-detect-the-chunk-language-when-setting-knitr-chunk-options
+knitr::opts_hooks$set(
+  echo = function(options) {
+    if (!is.null(options$echo)) {
```

Comment:
Isn't this setting the defaults only if the result is _not_ nulll?

Reply:
Yeap, that'd fix the problem I'm seeing.